js/server/process-vs.js

```diff
@@ -1,10 +1,18 @@
-const _ = require('lodash');
-const moment = require('moment');
-const { START_DATETIME, MULTIPLIER_MATURITY } = require('./config');
+const _ = require("lodash");
+const moment = require("moment");
+const { START_DATETIME, MULTIPLIER_MATURITY } = require("./config");
 
-const { processRewardBuckets } = require('./util/bucket-util');
-
-function processVSGlobalState (lastGlobalState, timestamp, eventsByUser) {
+const { processRewardBuckets } = require("./util/bucket-util");
+// const { getTimeIndex } = require("./util/getTimeIndex");
+const config = require("./config");
+/*
+  Filter out deposit events after config.DEPOSIT_CUTOFF_DATETIME,
```

Comment:
can remove if no longer in use?

Reply:
@musnit It's [in use](https://github.com/Sifchain/cryptoeconomics/pull/46/files/679504b02d81bcf08c035d0b0be29d78e70ab5d6#diff-fbe8325ccde4f8f78dcdbaf8818169ecfd7d04998a4851fc935ef3f9c2576140R88). ESLint would've caught it otherwise.

Reply:
@mccallofthewild ah sorry was referring to the commented out code below this

Reply:
Ah, yep. Was trying to appease the linter. Fixed in 66eb26a